cores/arduino/HardwareSerial.h

```diff
@@ -134,6 +134,10 @@ class HardwareSerial : public Stream
     using Print::write; // pull in write(str) and write(buf, size) from Print
     operator bool() { return true; }
 
+    void (*user_onReceive)(void);
```

Comment:
The function pointer member shouldn't be public, or is there a reason for it?

Reply:
No, there is no need for it.
As I wrote before: it is just copy/paste from Wire.
I'll change this and update PR.